.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Putting local environment IDE settings in source control is often objected. Is this common for VS Code projects?

Reply:
good point, i did it with the best intentions, but i see why it could be a problem, i'll remove it since it's not that important to have anyway